companies/octo-telematics.json

```diff
@@ -0,0 +1,28 @@
+{
+  "slug": "octo-telematics",
+  "relevant-countries": [
+      "it",
+      "es",
+      "de",
+      "gb",
+      "pl",
+      "us",
+      "br"
+  ],
+  "categories": [
+      "telecommunication",
+      "commerce"
+  ],
+  "name": "Octo Group s.p.a.",
+  "address": "Via Vincenzo Lamaro 51\n00173 Roma\nItalia",
+  "phone": "+39 06 726 5341",
+  "fax": "+39 06 729 0145",
+  "email": "dpo@octotelematics.com",
+  "web": "https://www.octotelematics.com/",
+  "sources": [
+      "https://www.octotelematics.com/",
+      "https://www.octotelematics.com/privacy-and-cookie-policy/"
+  ],
+  "request-language": "it",
```

Comment:
Does the company not respond to requests in other languages? They at least seem to offer their website in English, so I would just not set the `request-language`.